docs/source/hooks/common_use_cases.md

```diff
@@ -214,3 +214,77 @@ class MetadataHook:
         for dataset_name, dataset in catalog.datasets.__dict__.items():
             print(f"{dataset_name} metadata: \n  {str(dataset.metadata)}")
 ```
+
+## Use Hooks to debug your pipeline
+You can hooks to launch a [post-mortem debugging session](https://docs.python.org/3/library/pdb.html#pdb.post_mortem) with [`pdb`](https://docs.python.org/3/library/pdb.html) using [Kedro Hooks](../hooks/introduction.md) when an uncaught error occurs during a pipeline run. [ipdb](https://pypi.org/project/ipdb/) could be integrated in the same manner.
```

Comment:
I think we capitalise Hooks right? It's debatable whether we should, but historically we have done this.